script/metric/import_sources.rb

```diff
@@ -0,0 +1,40 @@
+require File.expand_path('../../../config/environment',  __FILE__)
+require 'colorize'
+require 'json'
+require 'byebug'
+require 'rubygems'
+require 'csv'
+
+file_name = 'script/metric/data/sources.csv'
+
+def create_company company
+  return if Card.exists? company
+  Card.create! name: company, type_id: Card::WikirateCompanyID
+end
+
+Card::Env[:protocol] = 'http://'
+Card::Env[:host] = 'http://wikirate.org'
+Card::Auth.current_id = Card.fetch_id 'Richard Mills'
```

Comment:
it would be nice to have a lot of this stuff handled once and only once.  eg, top of file could say `require 'wikirate_import_script_shared`.   Any time you find yourself cutting and pasting multiple lines from one file to another, you should consider whether that should happen.